### PR TITLE
feat: shift management docs refactor

### PR DESCRIPTION
**Pull request by Anurag810**
image

image

image

### CODE REVIEW

File: erpnext_documentation/www/docs/user/manual/en/human-resources/shift-management.md [hunk 1]
```diff
@@ -81,29 +81,49 @@ The date from which 'Auto Attendance' should start marking attendance. You shoul
...
+* **Employee Level:**
+Shift Request Approver can also be set Employee-wise in the employee master.
+
+
+ <img class="screenshot" alt="Shift Request Approvers" src="{{docs_base_url}}/assets/img/human-resources/employee-level-approvers.png">
+
 
-	<img class="screenshot" alt="Shift Assignment Calendar"
-	src="{{docs_base_url}}/assets/img/human-resources/shift-assignment-calendar.png">
+If Shift Request Approver are set at both Employee-level and Department-level, the Employee-level Shift Request Approver will be considered as the default Leave Approver in this case.
```
Comment (akshitakl) [inline erpnext_documentation/www/docs/user/manual/en/human-resources/shift-management.md:128]: Add Related Documents section

File: erpnext_documentation/www/docs/user/manual/en/human-resources/shift-management.md [hunk 2]
```diff
@@ -16,21 +16,21 @@ The Shift Type document allows you to define the different types of Shifts in yo
 To access Shift Type, go to:
 > Home > Human Resources > Shift Management > Shift Type
```
Comment (akshitakl) [inline erpnext_documentation/www/docs/user/manual/en/human-resources/shift-management.md:18]: Add pre-requisites section

**Review by akshitakl — CHANGES_REQUESTED** [review]

File: erpnext_documentation/www/docs/user/manual/en/human-resources/shift-management.md (hunk 1 above)
Comment (Anurag810) [reply erpnext_documentation/www/docs/user/manual/en/human-resources/shift-management.md:135]: There is no related Document section

File: erpnext_documentation/www/docs/user/manual/en/human-resources/shift-management.md (hunk 2 above)
Comment (Anurag810) [reply erpnext_documentation/www/docs/user/manual/en/human-resources/shift-management.md:18]: Shift Type is itself master there are no prerequisites, but added in shift request.

**Review by akshitakl — APPROVED** [review]

**Pull request merged**